projection/bridge_activity/bridge_pending_activity/bridge_pending_activity.go

```diff
@@ -34,37 +36,44 @@ type BridgePendingActivity struct {
 	logger  applogger.Logger
 
 	migrationHelper migrationhelper.MigrationHelper
+
+	listenedChannelIdToConfig map[ChannelId]*CounterPartyChainConfig
 }
 
 type Config struct {
-	ThisChainId           string `mapstructure:"this_chain_id"`
-	ThisChainName         string `mapstructure:"this_chain_name"`
-	CounterpartyChainName string `mapstructure:"counterparty_chain_name"`
-	ChannelId             string `mapstructure:"channel_id"`
-	StartingHeight        int64  `mapstructure:"starting_height"`
+	ThisChainName      string                    `mapstructure:"this_chain_name"`
+	CounterPartyChains []CounterPartyChainConfig `mapstructure:"counteparty"`
+}
+
+type CounterPartyChainConfig struct {
+	ChainName      string `mapstructure:"chain_name"`
+	ChannelId      string `mapstructure:"channel_id"`
+	StartingHeight int64  `mapstructure:"starting_height"`
```

Comment:
I think we missing the usage of this config, right?

Reply:
Good catch, updated the PR with the `StartingHeight` check